docs/src/man/tour.md

```diff
@@ -379,18 +431,24 @@ plot!(Singleton.(S), lab="S", c=:magenta)
 
 ## Polyhedral approximations
 
-In this library we consider representations of closed convex sets in the usual
-sense from convex geometry: A set is closed if it contains all its limit points.
-A set ``S`` is convex if for any ``m`` points ``v_j ∈ S`` and ``m`` non-negative
-numbers ``λ_j`` that sum up to ``1`` we have that ``∑_j λ_j v_j ∈ S`` as well.
-Alternatively, a closed convex set is an intersection of (possibly infinitely many)
-closed half-spaces.
+In this library we mainly consider representations of closed convex sets in the
+usual sense from convex geometry: A set is closed if it contains all its limit
+points. A set ``S`` is convex if for any ``m`` points ``v_j ∈ S`` and ``m``
+non-negative numbers ``λ_j`` that sum up to ``1`` we have that
+``∑_{j=1}^m λ_j v_j ∈ S`` as well. Alternatively, a closed convex set is an
+intersection of (possibly infinitely many) closed half-spaces.
+
+One of the central functions are `support_function(d, X)` (with the alias
+`ρ(d, X)` (type `\rho[TAB]`)) and `support_vector` (with the alias `σ` (type
+`\sigma[TAB]`)). The support function of a set ``X`` along direction
+``d ∈ \mathbb{R}^n`` corresponds to the (signed) distance of the maximum element
+in ``X`` along direction ``d``. The support vector is a (among possibly
+infinitely many) maximizer of the support function.
 
-One of the central functions are `support_function(d, X)` (which admits the alias `ρ(d, X)` (type `\rho[TAB]`))
-and `support_vector` (which admits the alias `σ` (type `\sigma[TAB]`)). The support function
-of a set ``X`` along direction ``d ∈ \mathbb{R}^n`` corresponds to the (signed) distance
-of the maximum element in `X`` along direction ``d``. The result of the support vector
-(among possibly infinitely many) that are maximizers of the support function.
+The support function also characterizes the half-space that tightly covers a set
+in a given direction. Hence it can be used to obtain a polyhedral
+(over-)approximation of any set. This topic is further covered in other sections
+of the manual.
```

Comment:
I added this paragraph to explain why the subsection is called "Polyhedral approximations".

Reply:
thanks. yes in fact we could add an example and plot to complement the section, as in the picture below (taken from [here](https://viterbi-web.usc.edu/~jdeshmuk/teaching/cs699-fm-for-cps/Papers/A3.pdf)):

![Screenshot from 2021-05-31 09-31-10](https://user-images.githubusercontent.com/12424594/120193553-f4e7a400-c1f2-11eb-87c6-5d5c47adc0b8.png)